chamber/commands/__init__.py

```diff
@@ -0,0 +1,60 @@
+from django.core.management.base import BaseCommand
+
+from chamber.importers import BulkCSVImporter, CSVImporter
+
+import pyprind
+
+
+class ProgressBarStream(object):
+    """
+    OutputStream wrapper to remove default linebreak at line endings.
+    """
+
+    def __init__(self, stream):
+        """
+        Wrap the given stream.
+        """
+        self.stream = stream
+
+    def write(self, *args, **kwargs):
+        """
+        Call the stream's write method without linebreaks at line endings.
+        """
+        return self.stream.write(ending="", *args, **kwargs)
+
+    def flush(self):
+        """
+        Call the stream's flush method without any extra arguments.
+        """
+        return self.stream.flush()
+
+
+class ImportCSVCommandMixin(object):
+
+    def handle(self, *args, **kwargs):
+        self.import_csv()
+
+
+class BulkImportCSVCommand(ImportCSVCommandMixin, BulkCSVImporter, BaseCommand):
+
+    def _pre_import_rows(self, row_count):
+        self.bar = pyprind.ProgBar(row_count, stream=ProgressBarStream(self.stdout))
```

Comment:
Logika progress baru je nyní v těhle metodách, aby to nestrašilo v kódu importerů, jak navrhoval @matllubos.